Cleanup of the main README (#6)
eclipse-uprotocol/.github: profile/README.adoc

* Same subscription API across all platforms, formalized as a service itself (uSubscription service) where the subscription flows across platforms and across different transports automatically
  ** Implementation may be custom to a given platform (not relying on generated code)

== How to Get Started? 

> 1. *Pick a Programming Language:* To build applications and services (uEs) that uses uProtocol, you'll first need to pick a programming languages, such as: https://github.com/eclipse-uprotocol/up-rust[Rust], https://github.com/eclipse-uprotocol/up-java[Java], https://github.com/eclipse-uprotocol/up-cpp[C++], https://github.com/eclipse-uprotocol/up-python[Python], and https://github.com/eclipse-uprotocol/up-kotlin[Kotlin]. Each library contains a `README` that will explain how to build and use uProtocol for their programming language.

Reviewer: Various suggestions, so paste the modified version rather than separately each minor change:

*Select a Programming Language:* To build applications and services (uEs) that use uProtocol, first pick a programming language, ...

> 2. *Pick a Transport:* Next you'll need to decide the communication middleware (transports) your uE will run on top of, some options are SOME/IP, MQTT, zenoh, android (binder), etc...  Check out the list of projects below to find a combination of language and transport that suits your needs, if you don't find one, we would love a contribution back :-).

Reviewer 2: ```suggestion
2. *Pick a Transport:* Next you'll need to decide the communication middleware (transports) your uE will run on top of, some options are Eclipse Zenoh, SOME/IP, MQTT 5, Android (Binder), etc...  Check out the list of projects below to find a combination of language and transport that suits your needs, if you don't find one, we would love a contribution back :-).
```

> 2. *Pick a Transport:* Next you'll need to decide the communication middleware (transports) your uE will run on top of, some options are Eclipse Zenoh, SOME/IP, MQTT 5, Android (Binder), etc...  Check out the list of projects below to find a combination of language and transport that suits your needs, if you don't find one, we would love a contribution back :-).

Reviewer: *Select a Transport:* Next choose the communication middleware (transports) your uE will connect to, some options are...

> 3. *Setup a Deployment:* Finally, to run your uE, you'll need set of additional core services running in your system. The core services, such as uSubscription, uDiscovery, etc... are required to complete the protocol and connect your application/service to other implementations (i.e. to connect your Android system with a Linux implementation, connect to the cloud, etc...). 

Reviewer: *Setup a Deployment:* Finally, to run your uE, you need to ensure that uProtocol core services are running in your system. The core services, such as uSubscription, uDiscovery, etc... are required to route events

> NOTE: We are working on step-by-step guides for the last step so that development and deployment of uProtocol uEs will be a simple as possible. 

Reviewer 2: ```suggestion
NOTE: We are working on step-by-step guides for the last step so that development and deployment of uProtocol uEs will be as simple as possible. 
```

> If you really want to know how it all works, feel free to browse the https://github.com/eclipse-uprotocol/up-spec[Specifications]

Reviewer 2: ```suggestion
If you want to know all the details of how it works, feel free to browse the https://github.com/eclipse-uprotocol/up-spec[Specifications]
```
